interpreter/cling/include/cling/Interpreter/Transaction.h

```diff
@@ -268,6 +272,16 @@ namespace cling {
       return const_reverse_nested_iterator(0);
     }
 
+    void addModules(clang::Module* M) {
```

Comment:
This function adds only one module, should use singular `Module(`, not `Modules(`.